autospec/tarball.py

```diff
@@ -501,6 +501,8 @@ def find_extract(tar_path, tarfile):
         extract_cmd, tar_prefix = build_unzip(tar_path)
     elif tarfile.lower().endswith('.gem'):
         extract_cmd, tar_prefix = build_gem_unpack(tar_path)
+    elif tarfile.lower().endswith('.jar'):
```

Comment:
Is this handling alternate to maven java build systems case?

Reply:
No, this is needed by the maven bin case to get the License info and description.